### PR TITLE
[Reviewer AJH] Retire BGCF routing in S-CSCF if not originating or terminating

### PR DESCRIPTION
**Pull request by steelyvoid**
See linked PR.

Do you want SAS datestamps to be bumped when I check in?

### CODE REVIEW

**Comment by steelyvoid** [comment]
https://github.com/Metaswitch/sprout/pull/1491

File: clearwater_sas_resource_bundle.yaml [hunk 1]
```diff
@@ -1254,6 +1254,12 @@ events:
       As these two URIs are aliases, terminating processing will continue and originating-cdiv processing will not be performed.
     level: 80
 
+  0x8100EB:
+    summary: "No application server chain for this initial request"
```
Comment (AlexHockey) [inline clearwater_sas_resource_bundle.yaml]: I'm not sure "application server chain" is something that most operators will understand. I would rephrase to say something like "There is no user to serve for this initial request".

File: clearwater_sas_resource_bundle.yaml [hunk 2]
```diff
@@ -1254,6 +1254,12 @@ events:
...
+    details: |
+      No originating or terminating services will be performed.
```
Comment (AlexHockey) [inline clearwater_sas_resource_bundle.yaml]: Worth mentioning that the request will be routed on using normal SIP routing rules?

**Comment by AlexHockey** [comment]
A couple of minor phrasing comments inline. Yes, could you bump the version number please?
